homeassistant/components/atag/__init__.py

```diff
@@ -188,24 +91,21 @@ async def async_unload_entry(hass, entry):
 class AtagEntity(Entity):
     """Defines a base Atag entity."""
 
-    def __init__(self, coordinator: AtagDataUpdateCoordinator, atag_type: dict) -> None:
+    def __init__(self, coordinator: AtagDataUpdateCoordinator, entity_id: str) -> None:
```

Comment:
`entity_id` is a reserved variable name for `Entity.entity_id`. Please rename this variable here to something else.